### PR TITLE
enable Test_DependencyEnable test for go

### PR DESCRIPTION
**Pull request by lievan**
## Description

`Test_DependencyEnable` is passing for go, but not enabled. This PR enables it.

### CODE REVIEW

File: tests/test_telemetry.py [hunk 1]
```diff
@@ -495,7 +495,7 @@ def test_app_started_product_disabled(self):
                 ), "Product information is present telemetry data on app-started event when all products are disabled"
 
 
-@released(cpp="?", dotnet="?", golang="?", java="1.7.0", nodejs="?", php="?", python="?", ruby="1.4.0")
+@released(cpp="?", dotnet="?", golang="1.49.0", java="?", nodejs="?", php="?", python="?", ruby="1.4.0")
```
Comment (cbeauchesne) [inline tests/test_telemetry.py:498]: You removed the java version, is it a typo ?

**Review by cbeauchesne — APPROVED** [review]

**Comment by cbeauchesne** [comment]
Hi @lievan,

Is this PR still relevant ?

**Comment by cbeauchesne** [comment]
@katiehockman gentle ping 😃

**Review by katiehockman — APPROVED** [review]

**Review by cbeauchesne — APPROVED** [review]

**Pull request closed without merging**